lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -110,6 +103,20 @@ def self.init_hypervisor(options)
           init_vmpooler
         end
       end
+
+      # Execute a task but capture stdout and stderr to a buffer
+      def self.with_captured_output
+        begin
+          old_stdout = $stdout.clone
+          old_stderr = $stderr.clone
+          $stdout = StringIO.new
```

Comment:
Is there a chance we might run into memory issues with this for extremely long test runs, or test runs with a lot of output? We should be wary of that when we are implementing `beaker run`.